packages/next/build/webpack-config.ts

```diff
@@ -68,7 +68,12 @@ function getOptimizedAliases(isServer: boolean): { [pkg: string]: string } {
     __next_polyfill__fetch: require.resolve('whatwg-fetch'),
     unfetch$: stubWindowFetch,
     'isomorphic-unfetch$': stubWindowFetch,
-    'whatwg-fetch$': stubWindowFetch,
+    'whatwg-fetch': path.join(
```

Comment:
Should this still terminate?

```suggestion
    'whatwg-fetch$': path.join(
```